docs/apache-airflow/best-practices.rst

```diff
@@ -91,13 +91,16 @@ Variables
 ---------
 
 You should avoid usage of Variables outside an operator's ``execute()`` method or Jinja templates if possible,
-as Variables create a connection to metadata DB of Airflow to fetch the value, which can slow down parsing and place extra load on the DB.
+as Variables create a connection to metadata DB of Airflow to fetch the value, which can slow down parsing and
+place extra load on the DB.
 
 Airflow parses all the DAGs in the background at a specific period.
-The default period is set using ``processor_poll_interval`` config, which is by default 1 second. During parsing, Airflow creates a new connection to the metadata DB for each DAG.
+The default period is set using ``processor_poll_interval`` config, which is by default 1 second.
+During parsing, Airflow creates a new connection to the metadata DB for each DAG.
```

Comment:
```suggestion
The default period is set using the ``processor_poll_interval`` config, which is 1 second by default.
During parsing, Airflow creates a new connection to the metadata DB for each DAG.
```

I think this reads better? It’s not related to this PR, but since we are modifying the line anyway, let’s improve it if we can.

Reply:
Yes, that does. Thanks!